lib/xebow/animation.ex

```diff
@@ -22,11 +22,12 @@ defmodule Xebow.Animation do
           type: type,
           tick: non_neg_integer,
           speed: non_neg_integer,
+          loop: non_neg_integer | -1,
```

Comment:
Your typing here reminded me of something I should have thought of sooner: What do you think about `loop: non_neg_integer | :infinitely`?  I realized the atom may be more self-documenting than `-1`, so I'm curious how you feel about it.

Reply:
Yes very good. I was debating this when implementing but thought -1 makes sense as well since its definitely an invalid `:loop` value. The atom reads much nicer in function head pattern matching though.

Changed the `:loop` field to use `:infinite` to represent dynamic animations.